microsoft/vscode
# fix: preserve process.execArgv

This ensures flags like `--prof` are passed down to the vs code process so we can profile everything.

To test this:
1. run `./lib/node --prof .`
2. in another terminal, run `ps -ejww`

You should see `--prof` next to every vs code process.



P.S. this approach leans on a pattern already in the codebase: https://github.com/microsoft/vscode/blob/06bd7ae776e2706a4aba2690d95788327dd16c43/src/vs/base/parts/ipc/node/ipc.cp.ts#L198-L200
